app/main.py

```diff
@@ -2,0 +1,30 @@
+class Animal:
+    alive = list()
```

Comment:
The `alive` list is a class variable, meaning it is shared among all instances of `Animal`. This could lead to unexpected behavior if not managed carefully, especially in a multi-threaded environment or if instances are created and destroyed frequently. Ensure this aligns with your design intentions.